tests/cli_tests/test_cli.py

```diff
@@ -7,5 +7,5 @@ def test_main():
     runner = CliRunner()
     result = runner.invoke(cli)
 
-    assert result.exit_code == 2
+    assert result.exit_code == 0
```

Comment:
@ziw-liu I found I needed to revert your earlier change to get the tests to pass. 

Any idea?

Reply:
e.g. see failing tests here: https://github.com/mehta-lab/waveorder/pull/446

Reply:
This behavior changed in click 8.2.0, but in this PR click 8.1.8 gets installed in CI. I think some other package that waveorder depends on pinned click after #448 merged.

Reply:
Can this be reverted so it doesn't roll break #450?

Reply:
reverted!